hi-ml-cpath/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -764,7 +765,7 @@ def _get_loss(module: DeepMILModule, batch: Dict, batch_idx: int) -> Tensor:
     limit = 2
 
     for batch_idx, batch in enumerate(dataloader):
-        if encoder_type == SwinTransformer_NoPreproc.__name__:
+        if "Resnet" not in encoder_type:
```

Comment:
why this check is changed/done? perhaps add an explanatory comment. Also, perhaps use positive check rather than negative "not in"?

Reply:
because Densenet also expects 224 input size, the pooling doesn't work with mock pathmnist tiles of 28x28 
We already have an explanatory comment down in the test